django_nitro_mailer/forms.py

```diff
@@ -16,6 +16,7 @@ class EmailAdminForm(forms.ModelForm):
     context = forms.JSONField(
         required=False, initial={}, help_text=gettext_lazy("JSON context for rendering the email content.")
     )
+    send_email = forms.BooleanField(required=False, initial=False, widget=forms.HiddenInput())
```

Comment:
A to po  co?

Reply:
żeby to było takim przyciskiem o
![Zrzut ekranu 2024-09-23 o 11 47 12](https://github.com/user-attachments/assets/86bfffdb-d9ad-493d-9c8e-e0a1f516bc93)